docs/admin/live-restore.md

```diff
@@ -0,0 +1,74 @@
+<!--[metadata]>
++++
+title = "Keep containers alive during daemon downtime"
+description = "How to keep containers running when the daemon isn't available."
+keywords = ["docker, upgrade, daemon, dockerd, live-restore, daemonless container"]
+[menu.main]
+parent = "engine_admin"
+weight="6"
++++
+<![end-metadata]-->
+
+# Keep containers alive during daemon downtime
+
+By default, when the Docker daemon terminates, it shuts down running containers.
+Starting with Docker Engine 1.12, you can configure the daemon so that containers remain
+running if the daemon becomes unavailable. The live restore option helps reduce
+container downtime due to daemon crashes, planned outages, or upgrades.
+
+## Enable the live restore option
+
+There are two ways to enable the live restore setting to keep containers alive
+when the daemon becomes unavailable:
+
+* When you start the Docker daemon, pass the `--live-restore` flag:
```

Comment:
should we swap the `--live-restore` and `daemon.json` options? (making `daemon.json` the preferred one?